ci/ci.sh

```diff
@@ -2,6 +2,7 @@
 export RUN_ID=$(node generate-id.js)
 echo "Run id: $RUN_ID"
 npm run launch:services
+sleep 5 # Give time for services to finish starting up before starting tests
```

Comment:
I ran into a similar situation with tests running before a service was ready. A hardcoded sleep ended up either being too long and wasteful, or not enough.

If there are "ready" logs to grep for, code like the following can be used to deterministically wait till the service(s) are ready.

`# Wait for service to initialize
until [[ "${ready}" != "" ]]; do
    docker logs service &> logs
    ready=$(cat logs | grep "Ready." | head -1)
    echo "Service not ready..."
    sleep 1
done
echo "Service ready!"`

Reply:
Yep, agreed this is a dirty hack.  The ideal solution would be that the promise returned when starting or stopping a ceramic node wouldn't resolve until startup/shutdown was well and truly complete.  We do our best on this already, but we've seen issues with this behavior from IPFS before, which we have less control over.  I'm not even sure that this is really the problem in the first place.  So I don't think it makes sense to spend a lot of time building a robust "wait for startup" / "wait for shutdown" strategy here, this is just a quick change to see if this even makes a difference in the stability of the test in the first place.